kokoro/scripts/macos/build.sh

```diff
@@ -20,6 +20,9 @@ set -e
 # Display commands being run.
 set -x
 
+# Fix dubious ownership issues on Kokoro.
+git config --global --add safe.directory /tmpfs/src/github/SPIRV-Tools
```

Comment:
Could you move this below the variables below and then change the directory to `$SRC`? Having an absolute path that does not seem connected to the rest of the script could lead to problems if things change later.

Reply:
Also, this seems like it should be a separate PR. It seems unrelated to the version genrule.